prow/jobs/custom/test-infra.yaml

```diff
@@ -105,6 +105,7 @@ presubmits:
   knative/community:
   - name: pull-knativeteam-groups-test
     decorate: true
+    optional: true   # TODO(upodroid@, chizhg@): remove this after https://github.com/knative/community/pull/905 is merged
     run_if_changed: "^groups/"
```

Comment:
As a follow up it would be good to know why the regexp didn't work 

It was blocking this PR
This PR hit the issue

This PR hit this issue: https://github.com/knative/community/pull/919/files